README.md

```diff
@@ -1,96 +1,61 @@
 # KubeVirt Ansible
 
-This repository provides collection of playbooks to
-- [x] [Install KubeVirt on existing OpenShift cluster](#install-kubevirt-on-existing-cluster)
-- [ ] Deploy Kubernetes cluster on given machines and install KubeVirt
-- [x] [Deploy OpenShift cluster on given machines and install KubeVirt](#deploy-kubernetes-or-openshift-and-kubevirt)
-- [x] [Provision resources, deploy cluster and install KubeVirt](#deploy-new-kubernetes-or-openshift-cluster-and-kubevirt-with-lago)
+This repository provides a collection of playbooks to
+- [x] [Install KubeVirt on an existing OpenShift cluster](#install-kubevirt-on-existing-cluster): `install-kubevirt-on-openshift.yml`
+- [ ] Deploy a Kubernetes cluster on given machines and install KubeVirt: `deploy-kubernetes.yml`
+- [x] [Deploy an OpenShift cluster on given machines and install KubeVirt](#deploy-kubernetes-or-openshift-and-kubevirt): `deploy-openshift.yml` 
```

Comment:
`deploy-openshift.yml` playbook doesn't install kubevirt, it deploys openshift on given machines. you need to run `install-kubevirt-on-openshift.yml` to install it.

Reply:
@alexxa Any updates on this one?

Reply:
@lukas-bednar @gbenhaim Fixed.